include/core/jcommon.h

```diff
@@ -44,13 +45,14 @@ G_END_DECLS
 G_BEGIN_DECLS
 
 // FIXME copy and use GLib's G_DEFINE_CONSTRUCTOR/DESTRUCTOR
-void __attribute__((constructor)) j_init (void);
-void __attribute__((destructor)) j_fini (void);
+void __attribute__((constructor)) j_init(void);
+void __attribute__((destructor)) j_fini(void);
 
-JConfiguration* j_configuration (void);
+JConfiguration* j_configuration(void);
 
-JBackend* j_object_backend (void);
-JBackend* j_kv_backend (void);
+JBackend* j_object_backend(void);
+JBackend* j_kv_backend(void);
+JBackend* j_db_backend(void);
```

Comment:
This contains unrelated formatting changes again.

Reply:
I'll remove those space-only changes